calico/datamodel_v1.py

```diff
@@ -44,6 +44,9 @@
 POLICY_DIR = VERSION_DIR + '/policy'
 PROFILE_DIR = POLICY_DIR + "/profile"
 
+# Key used for leader election by Neutron mechanism drivers.
+NEUTRON_ELECTION_KEY = VERSION_DIR + '/neutron_election'
```

Comment:
Not sure this should go in the versioned directory, it's plugin-specific and felix won't be expecting the compare-and-set operations.  How about putting it under `/calico/openstack/...`